src/OrdinaryDiffEq.jl

```diff
@@ -386,7 +386,8 @@ export MagnusMidpoint, LinearExponential, MagnusLeapfrog, LieEuler, CayleyEuler,
 
 export Rosenbrock23, Rosenbrock32, RosShamp4, Veldd4, Velds4, GRK4T, GRK4A,
     Ros4LStab, ROS3P, Rodas3, Rodas23W, Rodas3P, Rodas4, Rodas42, Rodas4P, Rodas4P2, Rodas5, Rodas5P,
-    RosenbrockW6S4OS, ROS34PW1a, ROS34PW1b, ROS34PW2, ROS34PW3, ROS34PRw
+    RosenbrockW6S4OS, ROS34PW1a, ROS34PW1b, ROS34PW2, ROS34PW3, ROS34PRw, ROS3PRL, ROS3PRL2,
+    ROS2PR, ROS2S, ROS3PR, SCHOLZ4_7
```

Comment:
I wasnt sure how to name it, because 4 and 7 doenst refer to the order. Instead in the paper of S. Scholz it is method number 4.7 (https://link.springer.com/article/10.1007/BF02259094) which is why Rang called it Scholz4_7 in his paper (https://leopard.tu-braunschweig.de/receive/dbbs_mods_00057194, page 29). 
Should I still call it Scholz47?

Reply:
Ah, I see. Yes, in that case `Scholz4_7` may be better 👍